src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -41,13 +43,15 @@ public function __construct($defaultContext = [], \DateTimeZone $timezone = null
         $this->defaultContext = [
             self::FORMAT_KEY => \DateTime::RFC3339,
             self::TIMEZONE_KEY => null,
+            self::FORCE_CONTEXT_TIMEZONE => false,
         ];
 
         if (!\is_array($defaultContext)) {
             @trigger_error('Passing configuration options directly to the constructor is deprecated since Symfony 4.2, use the default context instead.', E_USER_DEPRECATED);
 
             $defaultContext = [self::FORMAT_KEY => (string) $defaultContext];
             $defaultContext[self::TIMEZONE_KEY] = $timezone;
+            $defaultContext[self::FORCE_CONTEXT_TIMEZONE] = null === $defaultContext[self::TIMEZONE_KEY] ? true : false;
```

Comment:
No need for a ternary here, just

```php
$defaultContext[self::FORCE_CONTEXT_TIMEZONE] = null === $defaultContext[self::TIMEZONE_KEY];
```
should be enough.

Reply:
Thanks.  Removed.